lib/parallel_cucumber/worker_manager.rb

```diff
@@ -72,7 +72,8 @@ def start_workers
       end
       @results.inject do |seed, result|
         seed.merge(result) do |_key, oldval, newval|
-          (newval[:finish_time] > oldval[:finish_time]) ? newval : oldval
+          oldval.nil? && newval.nil? ? @logger.warn('Both oldval and newval are empty') : @logger.info("Picking most recent time of two, newval: #{newval}, oldval: #{oldval}")
+          (newval.fetch(:finish_time, -1) > oldval.fetch(:finish_time, -1)) ? newval : oldval
```

Comment:
Hey @artem-arustamov ! Thank you for contribution. please could you replace ternary operators with "a bit more readable" style? :-)
Thank you!